101-anf-nfs-volume/azuredeploy.json

```diff
@@ -0,0 +1,186 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "location": {
+      "type": "string",
+      "defaultValue": "[resourceGroup().location]",
+      "metadata": {
+        "description": "Same Location of resource group for all resources in the same deployment."
+      }
+    },
+    "netAppAccountName": {
+      "defaultValue": "test-anf-acc",
```

Comment:
will this work for all users (i.e. it doesn't need to be unique within the subscription or globally unique?)

Reply:
Name unique within a subscription only

Reply:
So it will work for all users

Reply:
if it's required to be unique within a subscription, then you can only deploy one instance of this template to a sub (using the defaultValues)...  so if that's required (not sure if that's what you meant because the scenario is odd) we need to make the defaultValue unique

Reply:
got it fixed. Now every resources that needs to be unique within a sub will have a different name and template can be deployed multiple times